core/src/main/java/com/yahoo/ycsb/generator/FileGenerator.java

```diff
@@ -36,20 +36,21 @@
 	 */
 	public FileGenerator(String _filename)
 	{
+	  filename = _filename;
 		try {
-			filename = _filename;
 			File file = new File(filename);
 			FileInputStream in = new FileInputStream(file);
 			reader = new BufferedReader(new InputStreamReader(in));
 		} catch(IOException e) {
-			System.err.println("Exception: " + e);
+		    throw new RuntimeException(e);
```

Comment:
Are these errors caught somewhere else? Is there a reason that exceptions were just printed instead of being thrown previously?